stablehlo/dialect/StablehloOps.cpp

```diff
@@ -1783,6 +1783,51 @@ LogicalResult ReduceOp::inferReturnTypeComponents(
                             inferredReturnShapes);
 }
 
+void ReduceOp::build(OpBuilder&, OperationState& odsState, ValueRange inputs,
+                     ValueRange initValues, DenseIntElementsAttr dimensions,
+                     TypeRange elementTypes) {
+  odsState.addOperands(inputs);
+  odsState.addOperands(initValues);
+  odsState.addAttribute(getDimensionsAttrName(odsState.name), dimensions);
```

Comment:
I think this will fail because `ReduceOp` expects `DenseI64ArrayAttr` now: https://github.com/openxla/stablehlo/blob/6661fb318a166c61ddde7867af473989bf227dc8/stablehlo/dialect/StablehloOps.td#L1475

Reply:
Thanks for pointing. PTAL.

Reply:
LGTM! For anyone reading this, when calling the builder you may need to do e.g. `rewriter.getDenseI64ArrayAttr` instead of `rewriter.getI64TensorAttr`.